examples/USGS/USGS_NLCD_RELEASES_2023_REL_RCMAP_V6_TRENDS.js

```diff
@@ -0,0 +1,27 @@
+// Import the NLCD RCMAP TRENDS image.
+var dataset = ee.Image('USGS/NLCD_RELEASES/2023_REL/RCMAP/V6/TRENDS');

```

Comment:
this should be ee.ImageCollection. But even fixing this, I don't see anything when I run this script because the source collection is empty

Reply:
When I comment out Line 2 and add in Line 3 (to my local - Source) it works fine.
(the Source is set to Anyone can read) 
var dataset = ee.Image('projects/usgs-gee-audra-griebel/assets/eros/rcmap_v6_trends/TRENDS');